v1.5 final release: quick hits (#3249)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/about-access.md

---
title: "About user access in dbt Cloud"
id: "about-user-access"
---

> :::info User access != Model access

Reviewer: ```suggestion
:::info User access is not Model access
```

> **User groups & access** are distinct from **model groups && access**, a language construct in dbt-core. For docs on the latter, see: ["Model access"](model-access).

Reviewer: ```suggestion
**User groups and access** and **model groups and access** mean two different things. "Model groups and access" is a specific term used in the language of dbt-core. Refer to [Model access](/docs/collaborate/govern/model-access)  for more info on what it means in dbt-core.
```

:::

> ## Overview

Reviewer: ```suggestion
```
